mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -32,13 +32,14 @@ CLASS none/sg net/minecraft/entity/Entity
 	FIELD as uuidString Ljava/lang/String;
 	FIELD at glowing Z
 	FIELD au passengerList Ljava/util/List;
-	FIELD av vehicle Lnone/sg;
+	FIELD av ridingEntity Lnone/sg;
```

Comment:
Was going to use that, both of them make sense but in hindsight ridingEntity sounds a bit too much like a boolean.